BUILDING.md

```diff
@@ -125,9 +125,23 @@ Install [Homebrew](https://brew.sh/) if not already installed.
 #### Install Required Packages
 
 ```bash
-brew install cmake pkg-config go@1.22 qt@5 jq nvm yarn
+brew install cmake pkg-config go@1.22 qt@5 jq nvm yarn protobuf
```

Comment:
This will be a `go@1.23` and `qt@6` very soon, just keeping it here as a reminder :)

Reply:
@caybro, maybe soon, but right now 1.23 gives build error(